### ISSUE TITLE
Add an option to follow operand in Data Dump

### ISSUE THREAD

**Issue by 10110111** [eteran/edb-debugger#190]
Sometimes analysis view's information is hardly useful in principle, e.g. when the instruction at `[ER]IP` is something like `fsave`. Thus it'd be nice to have an option to follow operand in Data Dump, which would switch currently active tab of Data Dump to the address corresponding to the operand.

**Comment by eteran** [comment]
If I understand this correctly, you'd like an option which when enabled will make the analysis able to update the data dump views to view the data being referenced by the current instruction. So stepping over several instructions that reference memory will have it hop from address to address?

I like it. I am envisioning a special tab which will follow the stream of addresses (so the user can still keep the tabs they are specifically interested in stable). I'll think on how to best present this in the UI

**Comment by 10110111** [comment]
This is an interesting interpretation, although I just wanted a context menu option of "Follow in Dump" like we currently have for Register View.

**Comment by eteran** [comment]
Ah, I see what you mean, (and it's a lot easier than what I was thinking). I'll get that in there. We already have the ground work laid out since jmps have code to "follow target".

**Comment by eteran** [comment]
I could have sworn that edb had this feature... and it DID! Unfortunately, it was temporarily broken by a minor off by one error int he Capstone wrapper code. As of my latest push, if you right click on an instruction with a constant as a parameter, there will be "Follow Constant in Dump" and "Follow Constant in Stack" menu options.

**Comment by 10110111** [comment]
Well, it does work for immediates, but this is far from all one might want. In particular this is true for `fsave` mentioned in the OP and any other instructions accessing memory:
````
mov eax,[ebx]
mov rax,[rip+0x234324]
mov eax,[0x324324]
````

**Comment by eteran** [comment]
Fair enough, we'll re-open this issue and update it to be an enhancement of the functionality
